ONSdigital/dp-dataset-api
# Feature/merging mongodb changes

### What

- Added component tests for put versions endpoint, specifically ensuring the correct kafka message is sent for a CMD or Cantabular dataset.
- Added support for real Kafka container during component test
- Retains ability to still use mocked kafka

### How to review

- Run make test-component and check the component tests run.
- Check the new features added in versions.feature are testing the endpoint.

### Who can review

- Anyone with component test knowledge.
